k8s/upgrade/src/plugin/constants.rs

```diff
@@ -71,7 +71,7 @@ pub(crate) const DEFAULT_IMAGE_REGISTRY: &str = "docker.io";
 pub(crate) const UPGRADE_JOB_IMAGE_TAG: &str = "develop";
 
 /// Upgrade job container image repository.
-pub(crate) const UPGRADE_JOB_IMAGE_REPO: &str = "openebs";
+pub(crate) const UPGRADE_JOB_IMAGE_REPO: &str = "niladrih";
```

Comment:
I've seen this many times, to avoid this ever getting checked in we probably should make this configurable via helm so ppl don't have to edit code..

```suggestion
pub(crate) const UPGRADE_JOB_IMAGE_REPO: &str = "openebs";
```